pkgs/applications/audio/mbrola/default.nix

```diff
@@ -26,6 +26,10 @@ let
     # required for cross compilation
     makeFlags = [ "CC=${stdenv.cc.targetPrefix}cc" ];
 
+    env = lib.optionalAttrs stdenv.cc.isGNU {
+      NIX_CFLAGS_COMPILE = "-Wno-error=implicit-function-declaration";
+    };
```

Comment:
As mentioned on Matrix by (I think) @emilazy, it would be great if there were tool assistance for notating parts of a derivation that should be attempted to be removed on next update.

@infinisil, maybe this is something that `nixpkgs-vet` could do in time... though upon further thought, it's closer to the job that ofBorg does.